integration-cli/docker_cli_login_test.go

```diff
@@ -14,8 +15,7 @@ func (s *DockerSuite) TestLoginWithoutTTY(c *check.C) {
 	cmd.Stdin = bytes.NewBufferString("buffer test string \n")
 
 	// run the command and block until it's done
-	if err := cmd.Run(); err == nil {
-		c.Fatal("Expected non nil err when loginning in & TTY not available")
-	}
+	err := cmd.Run()
+	c.Assert(err, checker.NotNil, check.Commentf("Expected non nil err when loginning in & TTY not available"))
```

Comment:
but it will get printed with any errors , I havn't make a test @MHBauer, just  find this at gocheck docs
"Note that if the comment is constant, a better option is to simply use a normal comment right above or next to the line, as it will also get printed with any errors"